index.html

```diff
@@ -0,0 +1,225 @@
+      integrity="sha384-ENjdO4Dr2bkBIFxQpeoTz1HIcje39Wm4jDKdf19U8gI4ddQ3GYNS7NTKfAdVQSZe"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.4/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+    <title>AI Brains</title>
+  </head>
+  <body>
+    <header class="hidden-desktop container-fluid p-0">
+      <div class="d-flex p-2">
+        <ul
+          class="container d-flex justify-content-end px-5 fs-6 m-0 gap-3 text-white"
+        >
+          <li><i class="bi bi-facebook"></i></li>
+          <li><i class="bi bi-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Log out</li>
+        </ul>
+      </div>
+      <div class="container-fluid bg-white d-flex justify-content-center">
+        <div class="container d-flex justify-content-between p-4 fs-6 m-0">
+          <a href="index.html"
+            ><img src="img/AIBrain.ico" alt="AI brain logo" width="30px" /> AI
+            <span>B</span>rains</a
+          >
+          <ul class="d-flex fs-6 align-items-center m-0 gap-3 p-0">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <!-- Main content -->
+    <main class="container-fluid p-0">
+      <!-- Intro section -->
+      <!-- Menu mobile -->
+      <div id="menu" class="hidden-mobile flex-column">
+        <div class="menu-mobile flex-column">
+          <button class="close-btn"><i class="bi bi-x-lg"></i></button>
+          <ul class="menu-options">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+      <img
+        src="img/bg-intro-min.png"
+        alt="AI background image"
+        class="bg-image"
+      />
+      <section class="intro-container d-flex flex-column gap-3 p-3 p-lg-5">
+        <button class="hidden-mobile menu-btn btn align-self-start p-2">
+          <img src="img/btn-menu.png" alt="menu mobile" width="24px" />
+        </button>
+        <span class="fs-3 mt-5 title">"Hello! Human world"</span>
+        <h1 class="mb-3 title fw-bold">AI brains Global Summit 2023</h1>
```

Comment:
- [ ] Great job working on the design so far but I think it would be great if you add more font size and font weight to the heading since in the given design it had more font size and font weight so kindly add it 💯🔥🎉

- Your design
![image](https://user-images.githubusercontent.com/58553711/235037254-c4e1048d-bc49-4a6e-a843-ec3b293f53a4.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/235037303-15d3dd7e-46a1-4899-968f-e619292f105d.png)